src/circuits/models.ts

```diff
@@ -66,7 +70,33 @@ export enum CircuitId {
   // AtomicQuerySig is a type for credentialAttrQuerySig.circom
   AtomicQuerySigV2 = 'credentialAtomicQuerySigV2',
   // AtomicQuerySigOnChain is a type for credentialAtomicQuerySigOnChain.circom
-  AtomicQuerySigV2OnChain = 'credentialAtomicQuerySigV2OnChain'
+  AtomicQuerySigV2OnChain = 'credentialAtomicQuerySigV2OnChain',
+
+  /**
+   * @beta
+   */
+  // AtomicQueryV3CircuitID is a type for credentialAtomicQueryV3.circom
+  AtomicQueryV3 = 'credentialAtomicQueryV3',
```

Comment:
change the value of constant AtomicQueryV3  to credentialAtomicQueryV3-beta.0
change the value of constant AtomicQueryV3OnChain  tocredentialAtomicQueryV3OnChain-beta.0 

constant name must remain v3 without beta